src/component/ChairList/ChairList.js

```diff
@@ -4,29 +4,31 @@ import styled from 'styled-components';
 
 import ChairCard from '../ChairCard/ChairCard';
 
-export default function ChairList({chairsData, toggleDescription, showDescription}) {
+export default function ChairList({chairs, toggleDescription, showDescription, setFavoriteChair, toggleLike}) {
```

Comment:
Please have a look at which props you don't use anymore (for example `setFavoriteChair`) and remove those :)